### PR TITLE
Generate Rails 6 applications with Sprockets 4-style assets

### PR DESCRIPTION
**Pull request by mjgiarlo**
Blacklight is currently generating Rails 6 applications with Sprockets 3-style assets. With this change, the `blacklight:server` task spins up a working server (with Sprockets 4-style assets) instead of screaming about webpacker.

Related to #2266, discovered in #2263.

### CODE REVIEW

File: tasks/blacklight.rake [hunk 1]
```diff
@@ -30,7 +30,10 @@ namespace :blacklight do
   end
 
   namespace :internal do
-    task seed: ['engine_cart:generate'] do
+    desc 'Index seed data in test ap'
+    task :seed do
+      Rake::Task['engine_cart:generate'].invoke unless File.exist?(EngineCart.destination)
```
Comment (mjgiarlo) [inline tasks/blacklight.rake]: This allows repeated runs of the blacklight server task with the same generated app.

File: tasks/blacklight.rake [hunk 2]
```diff
@@ -30,7 +30,10 @@ namespace :blacklight do
   end
 
   namespace :internal do
-    task seed: ['engine_cart:generate'] do
+    desc 'Index seed data in test ap'
```
Comment (jcoyne) [inline tasks/blacklight.rake]: typo

File: tasks/blacklight.rake (hunk 1 above)
Comment (jcoyne) [reply tasks/blacklight.rake]: Doesn't engine cart already check that the app is the same and do nothing?

File: lib/generators/blacklight/assets_generator.rb [hunk 3]
```diff
@@ -20,6 +20,15 @@ def create_sprockets_javascript
           //= require turbolinks
         CONTENT
       end
+
+      append_to_file 'app/assets/config/manifest.js', "\n//= link application.js"
```
[the remark this answers is not in the document]
Comment (mjgiarlo) [reply lib/generators/blacklight/assets_generator.rb]: I'm going to add a version check for Sprockets. Coming in a moment.

File: tasks/blacklight.rake (hunk 2 above)
Comment (mjgiarlo) [reply tasks/blacklight.rake]: good eye, thx! fixed

File: tasks/blacklight.rake (hunk 1 above)
Comment (mjgiarlo) [reply tasks/blacklight.rake]: It should. I saw it regenerate once, but perhaps things were in a weird state. Removing.

**Pull request merged**